src/routes/MetaDetails/styles.less

```diff
@@ -104,4 +109,20 @@
             background-color: @color-background-dark5-70;
         }
     }
+}
+
+.addon-modal-container {
```

Comment:
must be renamed to something else because the modal does not contain addon but meta extention